src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorReadyPublisherTest.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Test.Common;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+
```

Comment:
What be up with all the blank lines. Was this done by C# remove and sort or something?

Reply:
Dang it, I thought I cleaned all those. Something changed in the latest version of VS and now when it does a CodeAction or something to insert `using`s it sorts them like this with lines between top-level names. If anyone knows of a setting I can flip to turn that off I'd be much obliged.

Reply:
@CyrusNajmabadi is this a new thing with the C# analyzers?

Reply:
seems the .editorconfig for that is `dotnet_separate_import_directive_groups = false`, I've added it to our editorconfig and will fix any spaces using's I see in this PR.

Not sure why it suddenly started enforcing that though.